CHANGELOG.md

```diff
@@ -15,6 +16,86 @@ The below sections record the changes for each API version in each release as fo
 * for subsequent alpha or release-candidate API versions, the delta with respect to the previous pre-release
 * for a public API version, the consolidated changes since the release of the previous public API version
 
+# r2.1
+
+## Release Notes
+
+This **public release** contains the definition and documentation of
+* location-verification v2.0.0-rc.1
+* location-retrieval v0.4.0-rc.1
+* geofencing-subscriptions v0.4.0-rc.1
+
+The API definition(s) are based on
+* Commonalities v0.5.0
+* Identity and Consent Management v0.2.1
+
+**Changelog since r1.2**
+
+
+* Full Changelog with the list of PRs and contributors: https://github.com/camaraproject/DeviceLocation/compare/r1.2...r2.1
+
+## location-verification v2.0.0-rc.1
+
+### Added
+
+* Add MultiSIM section to info.description by @jlurien in https://github.com/camaraproject/DeviceLocation/pull/291
+
+### Changed
+
+* Decrease radius minimum to "1" for circle-area by @maxl2287 in #285
+* Error schemas updated with enums by @jlurien in #281
+* Some error status and codes have been updated by @jlurien in #281
+* Section with guidelines about how to identify the device from access token and request body has been updated by @jlurien in #281
+* Add pattern for x-correlator by @bigludo7 in #290
+* Update 429 error message by @bigludo7 in #290
+
+### Fixed
+* Update errormessage for unsupported device identifiers by @maxl2287 in #261
+* Add quote-marks for `lastLocationTime` - examples by @maxl2287 in #287
+
+## location-retrieval v0.4.0-rc.1

```

Comment:
Same as for location-verification, a hint about the breaking changes would be good